apps/vercel/frontend/src/components/config-screen/ContentTypePreviewPathSection/ContentTypePreviewPathSelectionRow/ContentTypePreviewPathSelectionRow.tsx

```diff
@@ -67,13 +77,17 @@ export const ContentTypePreviewPathSelectionRow = ({
     [contentTypes]
   );
 
+  const isPreviewPathInvalid = useMemo(() => {
+    return isAppConfigureCalled && !configuredPreviewPath && !!configuredContentType;
```

Comment:
Yeah this is all fairly readable as code but is doing my head in as far as why we consider it invalid under these conditions.

Reply:
Have removed one condition here to simplify it slightly!